pom.xml

```diff
@@ -65,6 +65,7 @@
     </modules>
 
     <properties>
+        <kafka.connect.storage.common.version>10.0.2</kafka.connect.storage.common.version>
```

Comment:
Essentially what you need is: 
```suggestion
        <kafka.connect.storage.common.version>${project.version}</kafka.connect.storage.common.version>
```

See if that works in the properties section.

Reply:
Hmm, scratch that. It won't work. I remembered. 
I checked the `mvn help:effective-pom`

Reply:
I changed this to match the project version (`10.0.3-SNAPSHOT`) and then ran a release dry run on it; the property was automatically updated to `10.0.3` for the tagged release, and `10.0.4-SNAPSHOT` for the commit immediately after.

I think all that's necessary for this to work is to change this line to use `10.0.3-SNAPSHOT` as the version, and it'll be automatically updated every time we do a release.

Reply:
I think that is our best path here +1